app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -865,8 +897,67 @@ module.exports = {
   async getProposalVotes(proposalId) {
     return state.votes[proposalId]
   },
-  async getProposalVote(proposalId, address) {
-    return state.votes[proposalId].find(vote => vote.voter === address)
+  async submitProposalVote({
+    proposal_id,
+    base_req: { name, sequence },
+    option,
+    voter
+  }) {
+    let results = []
+    let fromKey = state.keys.find(a => a.name === name)
+    let fromAccount = state.accounts[fromKey.address]
+
+    if (fromAccount == null) {
+      results.push(txResult(1, `Nonexistent account`))
+      return results
+    }
+    // check nonce
+    if (parseInt(fromAccount.sequence) !== parseInt(sequence)) {
```

Comment:
just an idea: we can probably refactor all this nonce and account checking. but not in the scope of this PR.

Reply:
Cool, can u open an issue with the details ?

Reply:
https://github.com/cosmos/voyager/issues/1551